src/test/java/io/stargate/sgv2/jsonapi/api/v1/VectorSearchIntegrationTest.java

```diff
@@ -1433,7 +1433,7 @@ public void insertVectorWithUnmatchedSize() {
           .then()
           .statusCode(200)
           .body("errors", is(notNullValue()))
-          .body("errors[0].message", endsWith("Expected vector of size 5, but received 3"));
+          .body("errors[0].message", endsWith("Mismatched vector dimension"));
```

Comment:
These changes don't seem unique to this bridge removal PR. Just curious, is this based on user feedback we chose to implement on this branch only?

Reply:
When vector size mismatched the error was thrown out from the bridge decoder. Cql throw different messages (more technical) for vector size mismatch. So the exception is intercepted and translated to this message in ThrowableToErrorMapper.java